### PR TITLE
DSYS-214: Design review fixes

### PR DESCRIPTION
**Pull request by lucianoratamero**
**Link to task:** https://labcodes.atlassian.net/browse/DSYS-214
**Staging app:** https://design-review-fixes--labstorybook-master.netlify.app/

This PR fixes z-index for banners and adds base styles for fieldsets and legends.

### CODE REVIEW

File: labsystem/scss/components/_lab-alerts.scss [hunk 1]
```diff
@@ -125,4 +125,4 @@ $alerts-line-height: 20px;
 
 .lab-alert__button {
   @extend .lab-alert__button;
```
Comment (falconedani) [inline labsystem/scss/components/_lab-alerts.scss:127]:
```suggestion
  @extend .lab-banner__button;
```

File: labsystem/scss/components/_lab-alerts.scss [hunk 2]
```diff
@@ -86,7 +86,7 @@ $alerts-line-height: 20px;
   position: -webkit-sticky;
 
   top: 0;
-  z-index: 1;
+  z-index: 2;
```
Comment (falconedani) [inline labsystem/scss/components/_lab-alerts.scss:89]: Wouldn't it be better to increase it so it'd never have overlaps again? Like 9 or so?

**Review by falconedani — CHANGES_REQUESTED** [review]
Just a few details.

File: labsystem/scss/components/_lab-alerts.scss (hunk 1 above)
Comment (lucianoratamero) [reply labsystem/scss/components/_lab-alerts.scss:127]: fixed! in fact, it was the other way around (lab-banner__button outside, lab-alert__button inside), but this led me to find a misuse of classes inside the Banner component. everything should be fixed now.

**Review by falconedani — APPROVED** [review]
🏆

**Pull request merged**